Skipped-URLs

cordoba27 opened the pull request:
Updated GUI
Added function to scrape URLs that were skipped in the initial scraping run

cordoba27 committed:
Changed GUI picture

Added new GUI picture with PDF and Scrape skipped URLs

cordoba27 committed:
Delete ARGUS_GUI.png

Delete ARGUS_GUI.png

cordoba27 committed:
Update README

Changed picture name

cordoba27 committed:
Update GUI

Added new button and function

cordoba27 committed:
Added new function

New function to scrape URLs that had to be skipped during the initial run

cordoba27 committed:
Updated start_crawl

Added distinction between normal and skipped URLs

datawizard1337 commented on ARGUS.py:
```diff
@@ -410,15 +410,16 @@ def start_scraping():
     byte_size = int(e13.get())*1000000    # convert from MB to B
     settings_txt.write(settings_file.format(e1.get(), e2.get(), e3.get(), e4.get(), e5.get(), e6.get(), e8.get(), e9.get(), e10.get(), e11.get(), byte_size, e14.get(), e15.get()))
     settings_txt.close()
-    scrapyd_txt = open(script_dir + r"/scrapyd.conf", "w", encoding="utf-8")
```
Warum das? Hat das vorher denn überhaupt funktioniert?

cordoba27 replied:
Funktioniert hat es, war aber meines Erachtens falsch, hat also einfach ein zweites File in einem anderen Ordner erstellt

datawizard1337 replied:
Also vorher lag die Datei weiter oben und du hast sie einfach zusätzlich woanders abgelegt?

cordoba27 replied:
Ne, als ich die relativen Pfade geändert hab, hab ich das wohl übersehen oder falsch gemacht. Ich überprüfe es aber nochmal

cordoba27 replied:
Ja, stimmt so

datawizard1337 commented on bin/start_crawl.py:
```diff
@@ -10,10 +10,10 @@
 script_dir_edit = str(script_dir)[:-4] 
 
-def start_crawl():
+def start_crawl(scraping_type):
```
Für was ist das gedacht?

cordoba27 replied:
Um normale Chunks und die Chunks aus den skipped URLs zu unterscheiden

datawizard1337 reviewed (commented):
Check mal die Comments

cordoba27 committed:
Changed kill_all_jobs.py

Replaced file name (as that had been changed at some point). So far, only the URL lists were deleted and the ARGUS chunks remained.
